clouddriver-api/clouddriver-api.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
   api("com.netflix.spinnaker.kork:kork-plugins-api")
   api("com.netflix.spinnaker.kork:kork-credentials-api")
 
+  implementation "com.netflix.frigga:frigga"
```

Comment:
I think it's ok to add frigga to clouddriver-api - it is lightweight (believe it is just Java) and stable.